pkg/karmadactl/promote/promote.go

```diff
@@ -127,6 +135,9 @@ type CommandPromoteOption struct {
 
 // AddFlags adds flags to the specified FlagSet.
 func (o *CommandPromoteOption) AddFlags(flags *pflag.FlagSet) {
+	flags.BoolVar(&o.AutoCreatePropagationPolicy, "auto-create-propagation-policy", true, "Create a PropagationPolicy automatically")
+	flags.StringVar(&o.PropagationPolicyName, "propagation-policy-name", "",
+		"The name of a PropagationPolicy that automatically created by command. If not specified, a random suffix will be added to the PropagationPolicy")
```

Comment:
```suggestion
		"The name of a PropagationPolicy that automatically created by command. If not specified, a suffix will be added to the PropagationPolicy")
```

The suffix is not generated random. It is a specific hash code.

Reply:
Yes, you are right, nice catch! Thank you.